ompi/mca/coll/basic/coll_basic_bcast.c

```diff
@@ -168,10 +163,14 @@ mca_coll_basic_bcast_lin_inter(void *buff, int count,
                                      MCA_PML_BASE_SEND_STANDARD,
                                      comm, &(reqs[i])));
             if (OMPI_SUCCESS != err) {
+                ompi_coll_base_free_reqs(reqs, rsize);
```

Comment:
@bosilca Replace rsize by i+1: no need to free all the requests since only i+1 of them have been filled